utils/securityJobSummary_test.go

```diff
@@ -6,184 +6,279 @@ import (
+										Condition: "Malicious package",
+										Packages:  map[string]int{"npm://lodash:1.0.0": 1},
+									},
+									{
+										Policy:    "cvss_score",
+										Condition: "cvss score higher than 4.0",
+										Packages:  map[string]int{"npm://underscore:1.0.0": 1, "npm://test:2.0.0": 1},
 									},
-									UniqueFindings: 6,
 								},
 							},
 						},
 						{
-							Target:     filepath.Join(wd, "application2"),
-							Violations: formats.TwoLevelSummaryCount{formats.ViolationTypeSecurity.String(): formats.SummaryCount{"High": 1}},
-							Vulnerabilities: &formats.ScanVulnerabilitiesSummary{
-								ScaScanResults: &formats.ScanScaResult{
-									SummaryCount:   formats.TwoLevelSummaryCount{"High": formats.SummaryCount{"Not Applicable": 1}},
-									UniqueFindings: 1,
-								},
-							},
+							Target:          filepath.Join(wd, "application2"),
+							CuratedPackages: &formats.CuratedPackages{PackageCount: 3},
 						},
-						{
-							Target: filepath.Join(wd, "dir", "application3"),
+					},
+				},
+				{
+					Scans: []formats.ScanSummary{{
+						Target: filepath.Join(wd, "application3"),
+						CuratedPackages: &formats.CuratedPackages{
+							PackageCount: 5,
+							Blocked: []formats.BlockedPackages{{
+								Policy:    "Aged",
+								Condition: "Package is aged",
+								Packages:  map[string]int{"npm://test:1.0.0": 1},
+							}},
 						},
 					}},
 				},
-				ScanCommandSummaryResult{
-					Section:          Curation,
-					WorkingDirectory: wd,
-					Results: formats.SummaryResults{Scans: []formats.ScanSummaryResult{
-						{
-							Target: filepath.Join(wd, "application1"),
-							CuratedPackages: &formats.CuratedPackages{
-								Blocked: formats.TwoLevelSummaryCount{
-									"Policy: Malicious, Condition: Malicious package":          formats.SummaryCount{"npm://lodash:1.0.0": 1},
-									"Policy: cvss_score, Condition:cvss score higher than 4.0": formats.SummaryCount{"npm://underscore:1.0.0": 1},
-								},
-								Approved: 4,
-							},
+			},
+		},
+		{
+			name:                "No vulnerabilities",
+			index:               commandsummary.BinariesScan,
+			expectedContentPath: filepath.Join(summaryExpectedContentDir, "no_vulnerabilities.md"),
+			args:                &ResultSummaryArgs{BaseJfrogUrl: testPlatformUrl},
+			content: []formats.ResultsSummary{{
+				Scans: []formats.ScanSummary{{
+					Target:          filepath.Join(wd, "binary-name"),
+					Vulnerabilities: &formats.ScanResultSummary{ScaResults: &formats.ScaScanResultSummary{ScanIds: []string{TestScaScanId}, MoreInfoUrls: []string{"https://test-url"}}},
+				}},
+			}},
+		},
+		{
+			name:                "Violations - Not defined",
+			index:               commandsummary.BinariesScan,
+			violations:          true,
+			expectedContentPath: filepath.Join(summaryExpectedContentDir, "violations_not_defined.md"),
+			args:                &ResultSummaryArgs{BaseJfrogUrl: testPlatformUrl},
```

Comment:
same comment on urls - let's try the test data to be a bit more realistic

Reply:
its little as possible but also try to cover most cases in code